ocs_ci/ocs/ui/base_ui.py

```diff
@@ -0,0 +1,178 @@
+
+        by_locator(str): Command that tells Selenium IDE which
+        GUI element needs to operate on.
+        text(str): Send text to element
+        type(By): Set of supported locator strategies
+        timeout(int): Looks for a web element repeatedly until timeout(sec) happens.
+
+        """
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((type, by_locator)))
+        element.send_keys(text)
+
+    def is_expanded(self, by_locator, type=By.XPATH, timeout=30):
+        """
+        Check whether an element is in an expanded or collapsed state
+
+        Args:
+            by_locator(str): Command that tells Selenium IDE which
+            GUI element needs to operate on.
+            type(By): Set of supported locator strategies
+            timeout(int): Looks for a web element repeatedly until timeout(sec) happens.
+
+        return:
+            (str): "true"-element expanded, "false"-element collapsed
+
+        """
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((type, by_locator)))
+        return element.get_attribute("aria-expanded")
+
+    def choose_expanded_mode(self, mode, by_locator, type):
+        """
+        Select the element mode (expanded or collapsed)
+
+        by_locator(str): Command that tells Selenium IDE which
+        GUI element needs to operate on.
+        type(By): Set of supported locator strategies
+        mode(str): Element mode (expanded or collapsed)
+
+        """
+        current_mode = self.is_expanded(by_locator=by_locator, type=type)
+        if mode != current_mode:
+            self.do_click(by_locator=by_locator, type=type)
+
+    def take_screenshot(self):
+        """
+        Take screenshot using python code
+
+        """
+        time.sleep(2)
+        self.cnt_screenshot += 1
+        filename = os.path.join(
+            self.screenshots_folder, str(self.cnt_screenshot).zfill(3)
+        )
+        self.driver.save_screenshot(filename)
+
+
+def login_ui(browser, headless=False, chrome_type=ChromeType.GOOGLE):
+    """
+    Login to OpenShift Console
+
+    Args:
+        browser(str): type of browser (chrome, firefox..)
+        headless(bool): True- without GUI, False- with GUI
+        chrome_type (ChromeType): Select chrome type (chrome, chromium..)
+
+    return:
+        driver(Selenium WebDriver)
+
+    """
+    logger.info("Get URL of OCP console")
+    console_url = run_cmd(
+        "oc get consoles.config.openshift.io cluster -o"
+        "jsonpath='{.status.consoleURL}'"
+    )
+    logger.info("Get password of OCP console")
+    password = get_kubeadmin_password()
+    password = password.rstrip()
+    if browser == "chrome":
+        logger.info("chrome browser")
+        chrome_options = Options()
+        chrome_options.add_argument("--ignore-ssl-errors=yes")
+        chrome_options.add_argument("--ignore-certificate-errors")
```

Comment:
What about add those also based on some configuration value? Maybe someone would like to test ssl / certificate and this case there is no way to not ignore it.